io.openems.edge.evcs.api/src/io/openems/edge/evcs/v2/api/PhaseRotation.java

```diff
@@ -0,0 +1,71 @@
+package io.openems.edge.evcs.v2.api;
+
+import io.openems.common.types.OptionsEnum;
+import io.openems.edge.evcs.v2.api.Evcs.ChannelId;
+
+public enum PhaseRotation implements OptionsEnum {
+    /**
+     * EVCS which use standard hardware connection configuration.
+     * 
+     * <p>
+     * L1 is connect to L1,...
+     */
+    L1_L2_L3(1, "L1_L2_L3", Evcs.ChannelId.CURRENT_L1, Evcs.ChannelId.CURRENT_L2, Evcs.ChannelId.CURRENT_L3), //
+    /**
+     * EVCS which use a rotated hardware connection configuration.
+     * 
+     * <p>
+     * L1 is connect to L2,...
+     */
+    L2_L3_L1(1, "L2_L3_L1", Evcs.ChannelId.CURRENT_L2, Evcs.ChannelId.CURRENT_L3, Evcs.ChannelId.CURRENT_L1), //
+    /**
+     * EVCS which use a rotated hardware connection configuration.
+     * 
+     * <p>
+     * L1 is connect to L3,...
+     */
+    L3_L1_L2(1, "L3_L1_L2", Evcs.ChannelId.CURRENT_L3, Evcs.ChannelId.CURRENT_L1, Evcs.ChannelId.CURRENT_L2);
```

Comment:
Is the value of 1 everywhere a typo? It should be what L1 is physically connected to.
```suggestion
public enum PhaseRotation implements OptionsEnum {
    /**
     * EVCS which use standard hardware connection configuration.
     * 
     * <p>
     * L1 is connect to L1,...
     */
    L1_L2_L3(1, "L1_L2_L3", Evcs.ChannelId.CURRENT_L1, Evcs.ChannelId.CURRENT_L2, Evcs.ChannelId.CURRENT_L3), //
    /**
     * EVCS which use a rotated hardware connection configuration.
     * 
     * <p>
     * L1 is connect to L2,...
     */
    L2_L3_L1(2, "L2_L3_L1", Evcs.ChannelId.CURRENT_L2, Evcs.ChannelId.CURRENT_L3, Evcs.ChannelId.CURRENT_L1), //
    /**
     * EVCS which use a rotated hardware connection configuration.
     * 
     * <p>
     * L1 is connect to L3,...
     */
    L3_L1_L2(3, "L3_L1_L2", Evcs.ChannelId.CURRENT_L3, Evcs.ChannelId.CURRENT_L1, Evcs.ChannelId.CURRENT_L2);
```

Reply:
Absolutely great. Good catch. This might have saved hours of debugging.